pyinfra/api/host.py

```diff
@@ -95,6 +95,14 @@ class Host(object):
     executing_op_hash = None
     nested_executing_op_hash = None
 
+    loop_position = 0
```

Comment:
Does it work with multiple embedded loops ? It seems using `host.loop` inside another `host.loop` is going to override the `loop_position`

Reply:
Great spot, definitely won't work, will fix...

Reply:
Fixed by https://github.com/Fizzadar/pyinfra/pull/876/commits/838a6f79362e81e9df91b3f871a67f010ff90634 & tested in https://github.com/Fizzadar/pyinfra/pull/876/commits/650b6fc5e9666b7ead8128b010d95a4579cbc6c2